front-end/src/screens/Footer/index.tsx

```diff
@@ -0,0 +1,54 @@
+import styled from '@xstyled/styled-components';
+import * as moment from 'moment';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Menu } from 'semantic-ui-react';
+
+interface Props {
+	className?: string
+}
+
+const Footer = ({ className }:Props ): JSX.Element => {
+	return (
+		<footer className={className}>
+			<Menu>
+				<Menu.Item>
+                    &copy; {moment.utc().year()} Parity Technologies
```

Comment:
indentation

Reply:
fixed in #347 - thanks for pointing out, need to fix my eslint vs code plugin